panic: logger cannot be nil

hi, I'm trying to run drive under docker. Getting this error:

```
/data # drive init aw
panic: logger cannot be nil

goroutine 1 [running]:
github.com/odeke-em/drive/src.New(0xc2080522c0, 0x0, 0x1)
	/go/src/github.com/odeke-em/drive/src/commands.go:204 +0x6d0
main.(*initCmd).Run(0xcd27a0, 0xc20800a020, 0x1, 0x1, 0xc20800ae40)
	/go/src/github.com/odeke-em/drive/cmd/drive/main.go:168 +0x53
github.com/odeke-em/command.Run()
	/go/src/github.com/odeke-em/command/command.go:202 +0xad
github.com/odeke-em/command.ParseAndRun()
	/go/src/github.com/odeke-em/command/command.go:209 +0x20
main.main()
	/go/src/github.com/odeke-em/drive/cmd/drive/main.go:120 +0x21e8

goroutine 5 [syscall]:
os/signal.loop()
	/usr/lib/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/lib/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 6 [chan receive]:
main.func·001()
	/go/src/github.com/odeke-em/drive/cmd/drive/main.go:1543 +0x64
created by main.initContext
	/go/src/github.com/odeke-em/drive/cmd/drive/main.go:1548 +0x2e3
```

I'm using image built using:
```
FROM blang/golang-alpine
RUN apk update && apk add build-base && go get -u github.com/odeke-em/drive/cmd/drive
WORKDIR /data
ENTRYPOINT /go/bin/drive
```

--- a/src/commands.go
+++ b/src/commands.go
@@ -170,7 +170,9 @@ func New(context *config.Context, opts *Options) *Commands {
 
 	var logger *log.Logger = nil
 
-	if opts != nil {
+	if opts == nil {
+		logger = log.New(stdin, stdout, stderr)
+	} else {
 		if opts.Quiet {
 			stdout = nil
 		}
@@ -198,10 +200,6 @@ func New(context *config.Context, opts *Options) *Commands {
 
 			opts.IgnoreRegexp = ignoreRegexp
 		}
-	}
-
-	if logger == nil {
-		panic("logger cannot be nil")
 	}
 
 	return &Commands{

